lint/linter.py

```diff
@@ -874,17 +874,17 @@ def build_options(self, options, type_map, transform=None):
 
                     options[name] = value
 
-    def get_chdir(self, settings):
+    def get_working_dir(self, settings):
         """Return the working dir for this lint."""
-        chdir = settings.get('chdir', None)
+        cwd = settings.get('working_dir', None)
 
-        if chdir:
-            if os.path.isdir(chdir):
-                return chdir
+        if cwd:
+            if os.path.isdir(cwd):
+                return cwd
             else:
                 persist.printf(
                     "{}: WARNING: wanted working_dir '{}' is not a dir"
```

Comment:
Not changed in this PR, but I suggest `is not a directory`.

Reply:
Done